.gitignore

```diff
@@ -32,4 +32,10 @@ yarn-error.log*
 .env
 
 # npm
-.npmrc
\ No newline at end of file
+.npmrc
+
+# direnv
+flake.nix
```

Comment:
I would say that these belong in your personal global gitignore, as it's not relevant to the project as such.